tests/test_error_wrappers.py

```diff
@@ -243,3 +244,24 @@ class Model(BaseModel):
 x
   field required (type=value_error.missing)"""
     )
+
+
+def test_nested_error():
+    class NestedModel1(BaseModel):
+        class NestedModel2(BaseModel):
```

Comment:
this a bit confusing, no need to nest the class definitions, just

```py
class NestedModel3(BaseModel):
    x: str

class NestedModel2(BaseModel):
    data2: List[NestedModel3]

class NestedModel1(BaseModel):
    data1: List[NestedModel2]
```

Reply:
done